src/main/resources/application.properties

```diff
@@ -8,4 +8,5 @@ server.error.path=/error
 spring-jpa.properties.hibernate.dialect=org.hibernate.dialect.PostgreSQLDialect
 spring-jpa.properties.hibernate.format_sql=true
 server.port=8080
-
+spring.security.user.name=dianamirlan@gmail.com
```

Comment:
there should never be personal information in a repository

also this is not used as I would suppose, so it should be removed